### ISSUE TITLE
Error when logging into steam, I don't get redirected

### ISSUE THREAD

**Issue by sbendeschii** [pepzwee/node-csgo-web-tradebot#135]
### Expected behavior
Able to sign in to steam

### Actual behavior
Signing in to steam takes me to the steam login page, however when i press "log in" it takes me back to the same exact page and says "error" and doesnt let me log in

### Steps to reproduce the behavior
No steamapis.com api key (yet) and no domain (yet)

### Other
1. [Operating System]: <W10>
2. [Node.js version]: <Latest>
3. [How did you start the script]: <pm2 start 0>

### Error Stack

Error

**Comment by iDefineHD** [comment]
Do, you have a working example of the error, as this is very bleak, also try https://steamapis.com/projects/node-csgo-web-tradebot if you have not already.

**Comment by sbendeschii** [comment]
Yes I do. The site is http://207.148.17.131/ (no domain yet, cause i don't know if it will work) If you try to sign in through steam, it says error at the bottom. 

Try it yourself. Also, yes, that is the tutorial I used to set it up.

**Comment by iDefineHD** [comment]
You have put the callback as www.youtube.com, thats why. you want it to direct back to your ip/domain.

**Comment by sbendeschii** [comment]
Yeah I only put youtube there because putting the ip of my site didn't do anything, just as a test to see if it would redirect. 

Here is the config.js, i deleted the keys for security purposes.



`'use strict'

module.exports = {
    appID: 730, // 730 - CS:GO
    contextID: 2, // ContextID
    bots: {
        bot_1: {
            siteName: 'Bot 1',  // Will be displayed under the "All bots" tab e.g. "Keys Only"
            accountName: '',    // bot_1 username
            password: '',       // bot_1  password
            twoFactorCode: '',  // shared_secret value
            identitySecret: '', // identity_secret value
            steamID64: '',  // SteamID64 of bot account can be found here: "https://steamid.io/"
            personaName: '',   // Nickname for bot account, will change on restart
        },
    },
    steamApiKey: '',    // Your Steam API key, get it here: https://steamcommunity.com/dev/apikey
    SteamApisKey: '',   // Your SteamApis.com key, get it here: https://steamapis.com
    SteamApisCompactValue: 'safe_ts.last_30d', // Use safe price calculated from 30 days of data, more info: https://steamapis.com/developers (Market Items - Optional Query Parameters "compact_value")
    site: {
        header: 'Dank.Trade', // Name/header/title of website. Prefix for  <title></title> (For more: /index.html line: 9) 
        steamGroup: '#',
        copyrights: 'Copyright © Dank.Trade 2018',  // Copyright text
    },
    domain: '207.148.17.131',    // Domain name only, follow the example (no http:// & no www & no /)
    website: 'http://207.148.17.131/',    // Website URL, follow the example (do not add / at the end)
    websitePort: 80,    // Website PORT, don't change it unless you're using a reverse proxy
    tradeMessage: 'Trade offer from csg0.trade | If you did not request this offer or the offer looks invalid please decline.', // Quite obvious
    rates: {
        ignoreItemsBelow: 0.05, // Ignore items below this price (price * rate < ignoreItemsBelow) - shows (Too Low) for user
        trashPriceBelow: 0.2,   // Items below this price are considered trash, the trash rate modifier will be applied
        // Items
        user: {
            key: 1,
            knife: 0.95,
            rare_skin: 0.95,
            weapon: 0.9,
            misc: 0.85,
            trash: 0.7,
        },
        bot: {
            key: 1.05,
            knife: 1,
            rare_skin: 1,
            weapon: 0.95,
            misc: 0.9,
            trash: 0.8,
        },
    },
}
`

**Comment by iDefineHD** [comment]
try 

> domain: '207.148.17.131', // Domain name only, follow the example (no http:// & no www & no /)
website: '207.148.17.131', // Website URL, follow the example (do not add / at the end) a messy paste. but lets see.

**Comment by sbendeschii** [comment]
Tried that earlier today and right now- Same thing. Error on sign-in. Don't know what to do honestly

**Comment by sbendeschii** [comment]
Any ideas? If anyone has the site working help me out

**Comment by sbendeschii** [comment]
hello?

**Comment by coooool123** [comment]
it is

**Comment by iDefineHD** [comment]
Did you even read the open issue that we spoke about? Dont post on old locked issues.
